test/interactions/panZoomInteractionTests.ts

```diff
@@ -1,39 +1,119 @@
+        let xScales = panZoomInteraction.xScales();
+
+        panZoomInteraction.xScales([]);
+        assert.deepEqual(panZoomInteraction.xScales(), [], "scales can be removed with the xScales call");
+
+        panZoomInteraction.xScales([xScale, xScale2]);
+        assert.deepEqual(xScales, panZoomInteraction.xScales(), "setting and adding x scales result in the same behavior");
+      });
+
+      it("can add the yScales in batch", () => {
+        let yScale2 = new Plottable.Scales.Linear();
+        panZoomInteraction.addYScale(yScale);
+        panZoomInteraction.addYScale(yScale2);
+        let yScales = panZoomInteraction.yScales();
+
+        panZoomInteraction.yScales([]);
+        assert.deepEqual(panZoomInteraction.yScales(), [], "scales can be removed with the yScales call");
+
+        panZoomInteraction.yScales([yScale, yScale2]);
+        assert.deepEqual(yScales, panZoomInteraction.yScales(), "setting and adding y scales result in the same behavior");
+      });
+
+      it("silently fails on adding an already existent xScale", () => {
+        panZoomInteraction.addXScale(xScale);
+        let oldXScaleNumber = panZoomInteraction.xScales().length;
+        panZoomInteraction.addXScale(panZoomInteraction.xScales()[0]);
+        assert.lengthOf(panZoomInteraction.xScales(), oldXScaleNumber, "the number of x scales is maintained");
+      });
+
+      it("silently fails on adding an already existent yScale", () => {
+        panZoomInteraction.addYScale(yScale);
+        let oldYScaleNumber = panZoomInteraction.yScales().length;
+        panZoomInteraction.addYScale(panZoomInteraction.yScales()[0]);
+        assert.lengthOf(panZoomInteraction.yScales(), oldYScaleNumber, "the number of y scales is maintained");
+      });
     });
 
     describe("Panning", () => {
+      let svg: d3.Selection<void>;
+      let SVG_WIDTH = 400;
+      let SVG_HEIGHT = 500;
+
+      let eventTarget: d3.Selection<void>;
+
+      let xScale: Plottable.QuantitativeScale<number>;
+      let yScale: Plottable.QuantitativeScale<number>;
+      let panZoomInteraction: Plottable.Interactions.PanZoom;
+
+      beforeEach(() => {
+        xScale = new Plottable.Scales.Linear();
+        xScale.domain([0, SVG_WIDTH / 2]).range([0, SVG_WIDTH]);
+        yScale = new Plottable.Scales.Linear();
+        yScale.domain([0, SVG_HEIGHT / 2]).range([0, SVG_HEIGHT]);
 
-      it("dragging a certain amount will translate the scale correctly (mouse)", () => {
+        svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+
+        let component = new Plottable.Component();
+        component.renderTo(svg);
+
+        panZoomInteraction = new Plottable.Interactions.PanZoom();
+        panZoomInteraction.addXScale(xScale);
+        panZoomInteraction.addYScale(yScale);
+        panZoomInteraction.attachTo(component);
+
+        eventTarget = component.background();
+      });
+
+      it("translates the scale correctly on draging (mouse)", () => {
```

Comment:
also, can you rewrite this test a little bit as it is not really readable right now
maybe use `deltaX` and `deltaY` to calculate the expected change in domain?